README.md

```diff
@@ -211,4 +213,22 @@ A list of currently active Blockchains projects. Sorted Alphabetically.
 
   #### Tutorials
    - [Complete Guide to Full Stack Solana Development by Nader Dabit](https://dev.to/edge-and-node/the-complete-guide-to-full-stack-solana-development-with-react-anchor-rust-and-phantom-3291)
- - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
\ No newline at end of file
+ - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
+ ----------
+ ## &#127916; youtube tutorials >> you can explore about crypto, blockchain,DAOs,EVM many more to learn...... &#11015;&#65039;
```

Comment:
Capitalize:
## &#127916; YouTube Tutorials

I think next bit should be on a different line underneath heading, I have changed it a little, what do you think of this?

Explore and learn about crypto, blockchain, DAOs, EVM and more...